data/brands/shop/supermarket.json

```diff
@@ -7522,12 +7523,16 @@
       }
     },
     {
-      "displayName": "О’кей",
+      "displayName": "О’КЕЙ",
       "id": "f546e4-d5eaac",
       "locationSet": {"include": ["ru"]},
+      "matchNames": ["окей", "ОКЕЙ"],
```

Comment:
Theres no need for matchNames for this: the matcher already takes capitalization into account
```suggestion
```

Reply:
> Theres no need for matchNames for this: the matcher already takes capitalization into account

I've fix it. Higly likely